frontend/src/components/bucket/BucketSidebar.vue

```diff
@@ -10,13 +16,41 @@ type Props = {
 
 const props = withDefaults(defineProps<Props>(), {});
 
+//Store
+const permissionStore = usePermissionStore();
+const userStore = useUserStore();
+
+const { getBucketPermissions } = storeToRefs(permissionStore);
+const { userSearch } = storeToRefs(userStore);
+
+// State
+const managedBy: Ref<string | undefined> = ref();
+
 // Emits
 const emit = defineEmits(['close-sidebar-info']);
 
 // Actions
 const closeSidebarInfo = async () => {
   emit('close-sidebar-info');
 };
+
+// Actions
+async function load() {
+  await permissionStore.fetchBucketPermissions({bucketId: props.sidebarInfo.bucketId});
+
+  const uniqueIds = [...new Set(getBucketPermissions.value
```

Comment:
when testing this PR in browser, I have 2 buckets: bucket1 only has me as manager. and nr-iit-showcase bucket.
if i open sidebar on bucket1 it shows just me as manager. if i then open sidebar on nr-iit-showcase bucket it again shows only me as manager. (if i refresh page  nr-iit-showcase shows all managers,)
`getBucketPermissions.value` is not pulling the latest state.
i wonder if it's because we created   `getBucketPermissions` const at the top of this file on line 23.
either that or we have a timing issue.